editor/components/block-list/block.js

```diff
@@ -218,7 +218,8 @@ export class BlockListBlock extends Component {
 		}
 
 		// Find all tabbables within node.
-		const tabbables = focus.tabbable.find( this.node );
+		const tabbables = focus.tabbable.find( this.node )
+			.filter( ( node ) => node !== this.node );
```

Comment:
I'm not sure when this could be the case? Following the logic, `find` will call `querySelectorAll`, which shouldn't include the source node.